src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -26,14 +37,15 @@ const CrossPodWriteForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
   const { selectedUser } = useContext(SelectUserContext);
+  const [expireDate, setExpireDate] = useState(null);
   const [username, setUsername] = useState();
```

Comment:
Looks like useState for username had not been initialized with anything. An empty string should do.